### ISSUE TITLE
comma-dangle > parsing error for multiline function call

### ISSUE THREAD

**Issue by jlsjonas** [eslint/eslint#11011]
**Tell us about your environment**

* **ESLint Version:** 4.19.1
* **Node Version:** 10.7.0
* **npm Version:** 5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    "root": true,
    extends: [
    	"airbnb-base",
    	"plugin:vue/recommended"
    ],
    plugins: [
        'vue',
    ],
    globals: {
        axios: true,
        route: true,
    },
    rules: {
        'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
        'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
        indent: ["error", 4],
        "vue/html-indent": 0,
        'vue/attributes-order': 0,
        "no-plusplus": 0,
        "no-debugger": 0,
        "no-underscore-dangle": ["error", { "allow": ["__"] } ],
        "no-param-reassign": 0,
        "no-mixed-operators": 0,
        "func-names": 0,
        "no-shadow": 0,
        "vue/max-attributes-per-line": 0,
        "no-return-assign": ["error", "except-parens"]
    },
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
            axios.get(
                route(...this.routeParams),
                { params: this.params },
            ).then(({ data }) => {
```

(automatically inside PHPStorm)
![image](https://user-images.githubusercontent.com/932193/47419815-4a2ac180-d77d-11e8-9dcd-ce225e303f32.png)

**What did you expect to happen?**
Eslint not to generate syntax errors (comma after `{ params: this.params }`)

**What actually happened? Please include the actual, raw output from ESLint.**
`Parsing Error: unexpected token )` after _fixing_ a comma-dangle

**Comment by Aladdin-ADD** [comment]
hi, thanks for the issue! seems you are using [trailing-function-commas](https://github.com/tc39/proposal-trailing-function-commas), it was introduced in es2017.  Can you try to set `ecmaVersion": 2017`(or 2018) to solve the issue?

**Comment by jlsjonas** [comment]
That did the trick; I guess `airbnb-base` or `plugin:vue/recommended` has a configuration issue
